wgpu/Cargo.toml

```diff
@@ -28,6 +28,10 @@ ignored = ["cfg_aliases"]
 
 [lib]
 
+[[test]]
+name = "no-gpu"
+required-features = ["noop"]
+
```

Comment:
I find `required-features` really annoying (need to remember to enable them). Lets make a `/tests/validation_tests` or something, which is a integration test part of the wgpu-test library, and can enable all the features it needs.

Reply:
Done.